packages/common/src/types/auth.ts

```diff
@@ -6,3 +20,41 @@ export enum AuthTokenPrefix {
     SERVICE_ACCOUNT = 'ldsvc_',
     PERSONAL_ACCESS_TOKEN = 'ldpat_',
 }
+
+type EmbeddedAccount = LightdashAuth<{
+    type: 'embed';
+    data: EmbedJwt;
+    source: string;
+}>;
+
+type LightdashUserAccount = LightdashAuth<{
+    type: 'session';
+    source: string;
+}>;
+
+type ServiceAccount = LightdashAuth<{
+    type: 'service-account';
+    source: string;
+}>;
+
+type LightdashAccountAuth =
```

Comment:
More consistent naming for these variables like `EmbeddedAccountAuth` - `LightdashUserAccountAuth` - `ServiceAccountAuth`